server.js

```diff
@@ -2040,6 +2040,135 @@ cache(function(data, match, sendBadge, request) {
   });
 }));
 
+// GitHub pullrequests integration.
+camp.route(/^\/github\/issues-pr(-raw)?\/([^\/]+)\/([^\/]+)\.(svg|png|gif|jpg|json)$/,
```

Comment:
Hmmm ... it'll really reduce the code, but increase the dependency relation between different functions.